lib/ide/parser/README.md

```diff
@@ -1,7 +1,7 @@
 You can run the tests by calling:
 ```
 sbt syntaxJS/fullOptJS
-wasm-pack test --headless --firefox --chrome --safari common/rust/parser
+wasm-pack test --headless --firefox --chrome --safari
```

Comment:
I think that this file can be removed altogether, as we already have a script that runs all the tests for all the packages (and otherwise this package is being tested like any else).
sbt command is not needed (or possible anymore, as the repos were split), as the JS build artifacts will be automatically downlaoded by the build.rs